dandischema/models.py

```diff
@@ -709,16 +736,21 @@ class RelatedParticipant(DandiBaseModel):
 
 
 class Participant(DandiBaseModel):
-    """Description about the sample that was studied"""
+    """Description about the Participant studied.
+
+    The Participant can be any individual or synthesized Agent. The properties
+    of the Participant refers to information at the timepoint when the Participant
+    engaged in the data being described.
+    """
 
     identifier: Identifier = Field(nskey="schema")
     altName: Optional[List[Identifier]] = Field(None, nskey="dandi")
 
     strain: Optional[StrainType] = Field(
-        None, description="Identifier for the strain of the sample", nskey="dandi"
+        None, description="Identifier for the strain of the participant.", nskey="dandi"
     )
     cellLine: Optional[Identifier] = Field(
-        None, description="Cell line associated with the sample", nskey="dandi"
+        None, description="Cell line associated with the participant.", nskey="dandi"
```

Comment:
I would use "participant or subject." in cases where it could be human or non-human

Reply:
i can do that.